blog/content/post/2020-01-15-GRUB2_and_3mdeb_minisummit.md

```diff
@@ -0,0 +1,177 @@
+---
+title: GRUB2 and 3mdeb minisummit 2019
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: piotr.krol
+layout: post
+published: true
+date: 2020-01-15
+archives: "2020"
+
+tags:
+	- grub2
+	- bootloader
+	- trenchboot
+categories:
+  - Firmware
+  - OS Dev
+  - Security
+
+---
+
+In December 2019 we had pleasure to meet with [Daniel Kiper](https://www.linkedin.com/in/dkiper/)
+GRUB maintainer, Software Developer, TrenchBoot technical leader at Oracle and
+conference speaker. Since we working with GRUB2 across many commercial projects
+and our customers leverage it often we decide to take advantage on this
+relation and organize meeting during which 3mdeb engineers and team leaders
+presented GRUB related topics. This blog post summarize discussion and create
+reference point for interested parties:
+
+Somehow "minisummit" name suvived since no one offered btter title after
```

Comment:
"_" or '_'  - let's be consequent.

Reply:
@lwcislo it would be way easier for me if you could submit suggestion like this:

```suggestion
Somehow 'minisummit' name survived since no one offered better title after
```